.github/build/action.yml

```diff
@@ -70,6 +99,9 @@ runs:
       shell: bash
       id: package-plugin
       run: |
+        # clean up previous build # TODO: think about better solution
```

Comment:
We could check out the repo two times and run the cloud build in one path and the free build in the other.

Reply:
adapted this one of the commits